cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -200,50 +101,275 @@
             }
         });
     }
+
+    // Expandable Overrides
+    .m-expandable_target {
+        padding: 0;
+        border: none;
+    }
+
+    .m-expandable_content-animated {
+        padding: 0;
+    }
+
+    .m-expandable_cue-label {
+        display: inline-block;
+    }
+
+    .m-expandable_header-left {
+        width: 73%;
+    }
+
+    .m-expandable_header-right {
+        width: 27%;
+    }
 }
 
-.node_expander {
-    position: absolute;
-    right: 10px;
-    top: 20px;
+.org-chart_branch {
+    /* Borders on .org-chart_branch pseudo elements (:after on first branch, :before on last)
+       draw lines connecting the branch headers to the line from .org-chart_root.
+    */
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 6 );
+        position: relative;
+        padding-top: @grid_gutter-width__em;
+        border-width: 0;
+
+        &:before,
+        &:after {
+            content: '';
```

Comment:
It's necessary if specifying width. Fixed in https://github.com/cfpb/cfgov-refresh/commit/3f65fc6c860c9e8fd00f10a506ea055b52a8b946.